charts/darts-portal/values.yaml

```diff
@@ -9,6 +9,15 @@ nodejs:
         - AppInsightsInstrumentationKey
         - redis-connection-string
         - darts-portal-session-secret
+        - name: AzureADTenantId
+          alias: TENANT_ID
+        - name: AzureADClientId
+          alias: CLIENT_ID
+        - name: AzureAdB2CClientId
+          alias: B2C_CLIENT_ID
+        - name: AzureAdB2CTenantId
+          alias: B2C_CLIENT_ID
+           
```

Comment:
Not sure we want to inject these secrets into our deployed app? What's this for?